feat: update subscription for pr #25394 (#293)
frappe/erpnext_documentation: erpnext_documentation/www/docs/user/manual/en/accounts/subscription.md

Visit the [Applying Discount](/docs/user/manual/en/selling/articles/applying-discount) page for more details.

### 3.5 Automatically create invoices

> Based on the [Subscription Plans](/docs/user/manual/en/accounts/subscription-plan) interval, invoices will be created automatically. Based on the Subscription Plans interval, invoices will be created automatically. They will be submitted automatically by default. If 'Submit Invoice Automatically' is disabled, the invoice will not be submitted automatically anymore.

Reviewer: ```suggestion
Based on the [Subscription Plans](/docs/user/manual/en/accounts/subscription-plan) interval, invoices will be created automatically. Based on the Subscription Plans interval, invoices will be created automatically. They will be submitted automatically by default. If 'Submit Invoice Automatically' is disabled, the invoice will be saved as a draft.
```
